typo.js

```diff
@@ -0,0 +1,30 @@
+/**
+ * @param {number} k
+ * @param {number} n
+ * @return {number[][]}
+ */
+let combinationSum3 = function (k, n) {
```

Comment:
:robot: OpenAI

The function `combinationSum3` is missing a `function` keyword.
```suggestion
let combinationSum3 = function(k, n) {
```

<!-- This is an auto-generated comment by OpenAI -->

Reply:
I don't think so

Reply:
:robot: OpenAI

@shoyuf: Thank you for your comment. You are correct, my previous suggestion was incorrect. I apologize for the confusion.

<!-- This is an auto-generated reply by OpenAI -->